### ISSUE TITLE
Consider increasing visual distinction between panes

### ISSUE THREAD

**Issue by juliasilge** [posit-dev/positron#2720]
We have gotten feedback multiple times from RStudio users that the Positron panes do not have enough visual distinction. I am actually surprised this isn't an accessibility issue for VS Code (but I am not an expert). @hadley finds this addition to `settings.json` an improvement:

```json
    "workbench.colorCustomizations": {
        "panel.border": "#333",
        "statusBar.border": "#333",
        "activityBar.border": "#333",
        "sideBar.border": "#333"      
    }
```

We could consider making this a default somewhere, or otherwise offering it as an easy option to choose.

**Comment by softwarenerd** [comment]
> I wonder if we should ship positron with a pair of default light and dark themes that resolve what we see as accessibility issues.

We do ship Positron with a pair of default Light and Dark themes.

<img width="608" alt="Image" src="https://github.com/user-attachments/assets/8e63db0f-70c6-494c-ad5f-d30c7e56cc29" />

So this is where we'd do this work.

**Comment by dhruvisompura** [comment]
The request for greater visual distinction between UI elements is exactly the kind of scenario the high contrast themes are designed to address. We also have support for customization of theme colors, so folks can tailor the experience to their specific preferences.

I'd be hesitant to change the default light theme. There are users who prefer the current appearance. Instead, a better path forward would be for users to customize colors locally or potentially introduce an additional theme (e.g. an "RStudio" variant).

**Comment by juliasilge** [comment]
Yes, that's the plan we're going to go with. In #7623 we did make some subtle changes that make the borders a smidge more prominent but we currently don't believe it's the right call to make the default more dramatic pane borders. A high contrast theme is a good option for people who prefer a lot of visual distinction between panes.

**Comment by hadley** [comment]
I tried the high contrast theme but I had to stop after my eyes started bleeding.
